### PR TITLE
Add documentation on how to add an external npm package

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/developer/client_side.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
+.. _developer_client_side:
+
+Client-side development
+=======================
+
+Usent an external librarie
```
Comment (asaunier) [inline doc/developer/client_side.rst:6]:
```suggestion
Using an external library
```

File: doc/developer/client_side.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+For one Javascript file in the package
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add in the alias of the file ``geoportal/webpack.apps.js.mako``:
+
+ .. code:: javascript
+
+    alias: {
+      ...,
+      '<your_package_name': path.resolve(__dirname, 'node_modules/<package_name>/<the_javascript>.js
```
Comment (asaunier) [inline doc/developer/client_side.rst:47]:
```suggestion
      '<your_package_name>': path.resolve(__dirname, 'node_modules/<package_name>/<the_javascript>.js
```

File: doc/developer/client_side.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+For multiple Javascript file in the package
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add in the alias of the file ``geoportal/webpack.apps.js.mako``:
+
+ .. code:: javascript
+
+    alias: {
+      ...,
+      '<your_package_name': path.resolve(__dirname, 'node_modules/<package_name>/<src_folder?>
```
Comment (asaunier) [inline doc/developer/client_side.rst:66]:
```suggestion
      '<your_package_name>': path.resolve(__dirname, 'node_modules/<package_name>/<src_folder>
```
=> I guess the trailing "?" is a typo?

File: doc/developer/client_side.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+And use it with:
+
+.. code:: javascript
+
+  import '<your_package_name>';
+
+
+For multiple Javascript file in the package
```
Comment (asaunier) [inline doc/developer/client_side.rst:57]:
```suggestion
Importing multiple Javascript files from the package
```

File: doc/developer/client_side.rst [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+In the ``docker-compose.override.sample.yaml`` file add in the ``webpack_dev_server`` service:
+
+.. code:: yaml
+
+   volumes:
+     - ./geoportal/node_modules: /app/node_modules
+
+
+For one Javascript file in the package
```
Comment (asaunier) [inline doc/developer/client_side.rst:38]:
```suggestion
Importing a single Javascript file from the package
```

File: doc/developer/client_side.rst (hunk 3 above)
Comment (sbrunner) [reply doc/developer/client_side.rst:66]: No, I add it because it's optional...

**Comment by sbrunner** [comment]
@asaunier  Updated :-)

File: doc/developer/client_side.rst (hunk 3 above)
Comment (asaunier) [reply doc/developer/client_side.rst:66]: There is a missing trailing ``>`` anyway

File: doc/developer/client_side.rst [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+Importing multiple Javascript files from the package
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add in the alias of the file ``geoportal/webpack.apps.js.mako``:
+
+ .. code:: javascript
+
+    alias: {
+      ...,
+      '<your_package_name': path.resolve(__dirname, 'node_modules/<package_name>/<src_folder?>
```
Comment (asaunier) [inline doc/developer/client_side.rst:66]:
```suggestion
      '<your_package_name>': path.resolve(__dirname, 'node_modules/<package_name>/<src_folder?>
```

**Pull request merged**